spec/icd/api/client_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+RSpec.describe Icd::Api::Client do
+  let(:client) do
+    described_class.new(client_id: '',
+                        client_secret: '')
+  end
+
+  context 'Fetch by Code -- Good Value' do
```

Comment:
What'a good value? From how I understand it, the value after the dot is sued to further specify the disease though it is not required for a code to be valid?

```suggestion
  context 'Fetch by Code -- Good Value' do
```

Reply:
Good question. Hey @KiotoJ 👋🏼 , can you remember? 🙇🏼

Reply:
Will address this in a follow up PR